src/utilities/coerceInputValue.js

```diff
@@ -126,7 +126,7 @@ function coerceInputValueImpl(
         fieldValue,
         field.type,
         onError,
-        addPath(path, field.name),
+        addPath(path, field.name, undefined),
```

Comment:
Actually here you have access to `type.name` so you can put it in the `Path`.

Reply:
I considered that, but I'm not sure that inputs should set `parentType`: since we don't know what the polymorphic input solutions are going to be yet, letting people rely on `parentType` existing is risky. Besides, the path is only used in errors and that doesn't currently use `parentType`, so adding it offers little or no benefit, IMO.

If you're sure, I'm happy to add it, but I was deliberate in not adding it to input types.

Reply:
I've added it as the final commit so you can pop it or add it as you see fit.

Reply:
@benjie Any input union solution will result in exactly one `parentType` if it's not like that we would have a bigger breaking change anyway.
Leaving `parentType` unset will be confusing for people accessing path.

Reply:
And it would be mean we will need two separate types for execution path and for path inside input values.

Reply:
Scratch it, it's not worth to add functionality that no one asked for just to unify types.
It actually makes sense to have separate `ExecutionPath` type just for execution.

Reply:
Thinking more about it, it's actually useful to be able to reference input field definition as part of the error message. So yes let's keep it.
Sorry for all the back and forth.

Reply:
No worries; it's valuable to see your thought process. I'll keep it :+1: